### DISCUSSION TITLE
Institution - CDE 12662779

### DISCUSSION THREAD

**Discussion by GeoffLyle** [CBIIT/ccdi-federation-api#67] [Metadata Field Ratification]
caDSR Link: [12662779](https://cadsr.cancer.gov/onedata/dmdirect/NIH/NCI/CO/CDEDD?filter=CDEDD.ITEM_ID=12662779%20and%20ver_nr=1)

Definition: "A sequence of characters used to identify, name, or characterize the laboratory, institute, or consortium that provided the information."

Note: This CDE has a list of institutions and the sequence of characters to identify them.  It does not contain an "Other" or "Not Reported" permissible values.

**Comment by Ssandor13** [comment]
Hi @jknable, I wanted to kindly ask if we want update this CDE to annotate as "St. Jude Children's Research Hospital" rather than "Saint Jude Children's Research Hospital" to keep it consistent with how the institution writes out it's name? Thank you.

**Reply by jknable** [reply]
This change has been made.

**Reply by Ssandor13** [reply]
Thank you so much!

**Reply by Ssandor13** [reply]
@jknable could we also update the definition to also reflect the "St." annotation? Thank you very much.

**Reply by jknable** [reply]
I have submitted that request to our terminology group and I am waiting on their response.  I will keep you advised.

**Reply by jknable** [reply]
The terminology group has agreed to update the concept and we have updated the permissible value in the Institution CDE.

**Reply by GeoffLyle** [reply]
@jknable 
I'm seeing two **SJCRH** values in the list of PVs.  
One with VM Public ID 12935205 has the definition "The Saint Jude Children's Research Hospital received its NCI designation in 1977 and was awarded status as a comprehensive cancer center by NCI in 2008. Research is focused specifically on childhood cancers, acquired and inherited immunodeficiencies and genetic disorders."
Other with VM Public ID 14793809 has the definition "The St. Jude Children's Research Hospital received its NCI designation in 1977 and was awarded status as a comprehensive cancer center by NCI in 2008. Research is focused specifically on childhood cancers, acquired and inherited immunodeficiencies and genetic disorders."

It looks like when the new PV was added, the old one wasn't removed.  I think it should be since there are now duplicate values for **SJCRH**

**Reply by Ssandor13** [reply]
Ah, yes, I do see the duplication now. I saw that 12935205 was updated for the VM long name, hence my request to update the definition which still says "Saint" but with that, the 14793809 record was created. @jknable

**Reply by jknable** [reply]
This has been updated and only the VM (ID 14793809) should be displaying.

**Comment by calkinsh** [comment]
This works for CBTN/CHOP

**Comment by hbeale** [comment]
What does "the information" mean in "...provided the information"? It could the organization that is hosting the API, the organization that generated the processed genomic data, the organization that is the source of the sequence data, or the organization that is the source of the biological tissue, or the organization that assembled the clinical data. 

I suspect we'll need a Treehouse VM. If so, I propose 
* Permissible Value = Treehouse
* VM Long Name = Treehouse Childhood Cancer Initiative at University of California Santa Cruz

**Reply by jknable** [reply]
@hbeale @jknable - can you please provide a sentence to describe this proposed permissible value?  We will need that to request a new term to be created for your organization. 

The CDE cited is intended to identify the organization/group which submits data; it is an identifier assigned to a data contributor.

**Reply by GeoffLyle** [reply]
@jknable 
Can you add this definition to our permissible value:

**Permissible Value** = Treehouse
**Long Name** = Treehouse Childhood Cancer Initiative at University of California Santa Cruz
**Definition** = A research group at the University of California Santa Cruz that aggregates pediatric cancer genomic data and uses computational analysis of genomic data to identify less toxic or more effective treatments.

**Reply by jknable** [reply]
I have sent for a new term.  When received we will add Treehouse to the CDE.

**Reply by jknable** [reply]
The terminology group has created the following concept.  We will add to the CDE with a permissible value of "Treehouse".  The Concept name and definition will be associated with it.

NCI Concept: Treehouse Childhood Cancer Initiative
NCI Code: C204768
Synonym: Treehouse Childhood Cancer Initiative at UC Santa Cruz; **Treehouse**
Definition: An initiative to find existing cancer drugs that might be applicable to childhood cancers. The group is analyzing the genomic data of children with cancer in the context of large data sets of both pediatric and adult cancers. This should identify situations where an available drug is predicted to work for a given patient’s tumor based on a similar genetic basis for the cancer.

**Comment by GeoffLyle** [comment]
From March 14, 2024 meeting:

Treehouse working on adding a PV for their group. May need to add other groups as need arises. Implementation question on how we can include this CDE while also ensuring that is does not preclude new Federation members from joining. If a quick onboarding process is in place to readily add a new PV for a new group, this should not be a major issue.​

Accepted on condition that we have an email contact for someone from the caDSR for new institutions who want to be added to this CDE.  Will work with curation team to add contact.

**Reply by jknable** [reply]
New organizations can be added to this CDE.  All we need is the acronym/short name, the full org name and a description like you provided for Treehouse.  Requests can be sent to me (janice.knable@nih.gov) or Maureen (maureen.ryan2@nih.gov).  It takes one business day to add a group to the CDE once we have a term for the group created which typically takes 2-3 business days.
